largescale/run-million-collection-test.sh

```diff
@@ -10,7 +10,7 @@ MONGO_TESTS_REPO="https://github.com/wiredtiger/mongo-tests"
 POCDRIVER_BRANCH="mongodb-million-collections"
 TEST_DIR="mongo-million-collection-test"
 BIN_DIR="build/install/bin/"
-PERF_MAKE_FLAGS="-j 20"
+PERF_MAKE_FLAGS="-j 16"
```

Comment:
Where did this number come from?

Reply:
Can use `multiprocessing.cpu_count()` to determine cores/threads for the system ?

Reply:
Can use `multiprocessing.cpu_count()` to determine cores/threads for the system ?

Reply:
@tammybailey got the number from running nprocs from the machine. Sounds like using `multiprocessing.cpu_count()` might be a better idea here.

Reply:
I realised that we can't use `multiprocessing.cpu_count()` because it is a bash script. I went back to using `nproc`

Reply:
Ahh gee! I did not see it is a bash script. facepalm!

Reply:
@jiechenbo did you run a patch build for the test to make sure the command works as expected?

Reply:
i haven't with the new changes. Just ran the patch build, the link is inside jira and verifies that it works